inmanta.spec

```
@@ -246,6 +247,9 @@ getent passwd inmanta >/dev/null || \
 exit
 
 %changelog
+* Tue Jan 11 2022 Arnaud Schoonjans <arnaud.schoonjans@inmanta.com> - 2021.2.2
```

```suggestion
* Tue Jan 11 2022 Arnaud Schoonjans <arnaud.schoonjans@inmanta.com> - 2022.1
```
I already bumped the version on master.